src/StripeResource.ts

```diff
@@ -356,15 +366,19 @@ StripeResource.prototype = {
     sleepSeconds = Math.max(initialNetworkRetryDelay, sleepSeconds);
 
     // And never sleep less than the time the API asks us to wait, assuming it's a reasonable ask.
-    if (Number.isInteger(retryAfter) && retryAfter <= MAX_RETRY_AFTER_WAIT) {
+    if (
+      retryAfter &&
```

Comment:
Why did we introduce this first condition here? This is subtly different:

For `retryAfter = 0`:
- `Number.isInteger(retryAfter) == true`
- `retryAfter && Number.isInteger(retryAfter) == false`

Reply:
That was to make sure `retryAfter` is not null. It actually seems like that was not necessary (there was an error there earlier but it doesn't show up when I remove the check). Thank you for flagging!